src/TurnService.ts

```diff
@@ -0,0 +1,71 @@
+import Color from './color';
+
+class TurnService{
+
+  public getThreeColor(side: Color[], sideVals: number[]): any{
+    return sideVals.map((val) => side[val])
```

Comment:
```suggestion
    return sideVals.map(val => side[val])
```
You can skip () when u have only 1 argument and u don't specify its data type